controllers/index.go

```diff
@@ -1,15 +1,28 @@
 package controllers
 
 import (
+	"log"
 	"net/http"
+	"reakgo/models"
 	"reakgo/utility"
 )
 
 func BaseIndex(w http.ResponseWriter, r *http.Request) {
-    name := []string{"Test1", "Test2"}
-    utility.RenderTemplate(w, r, "index", name)
+	name := []string{"Test1", "Test2"}
+	utility.RenderTemplate(w, r, "index", name)
+}
+
+func BaseAPI(w http.ResponseWriter, r *http.Request) {
+	log.Println(r.Header.Get("tokenPayload"))
+	err := models.VerifyToken(r)
```

Comment:
can we replace this part of code on
how to retrieve data from cache `utility.Cache.Get` with example ie: id, username print

 because of other developers also rewrite same think :rofl:

Reply:
Question isn't clear

Reply:
i can say because of here we are again checking token,
can we remove this and replace with get value from cache because of developer easily get value from cache if needs

Reply:
Yes, Sorry I didn't understand this earlier. Have fixed in my latest commit. Was an oversight from my end